templates/observability-remote-write-proxy-route.yml

```diff
@@ -0,0 +1,23 @@
+# This file contains an OpenShift template that creates a route helping to
+# directly access the Observability Remote Write Proxy in the
+# development environment.
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+name: observability-remote-write-proxy-route
+metadata:
+  name: observability-remote-write-proxy
+objects:
+- apiVersion: route.openshift.io/v1
+  kind: Route
```

Comment:
The addition of a separate route than the one used by KFM is intentional. Details around the reason and motivations why this has been done are available in the https://issues.redhat.com/browse/MGDSTRM-10333 Jira issue